zetta_utils/db_annotations/precomp_annotations.py

```diff
@@ -57,7 +58,9 @@ def __init__(self, line_id: int, start: Sequence[float], end: Sequence[float]):
         :param start: A tuple of three floats representing the start coordinate (x, y, z).
         :param end: A tuple of three floats representing the end coordinate (x, y, z).
 
-        Coordinates are in units defined by "dimensions" in the info file.
```

Comment:
btw, is there a reason we're not using `attrs` here like we do in the rest of the codebase?

Reply:
Just that I'm still not super familiar with it, or the reasons one might want to use it.

Reply:
It's just a cleaner more modern way to use python that reduces boilerplate and comes with a lot of free gooddies

Reply:
We are trying to use it for all classes except for the ones that inherent from pytorch.nn.module (pytorch does some incompatible black magic)

Reply:
OK.  I know you were eager to get this to your web team, so it's up to you whether you want to hold this until I can do that refactoring (probably tomorrow), or merge this as-is and I'll do another little PR to add `attrs`.